dspace-api/src/test/java/org/dspace/core/ContextModeIT.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.core;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import org.dspace.AbstractIntegrationTestWithDatabase;
+import org.dspace.authorize.ResourcePolicy;
+import org.dspace.authorize.factory.AuthorizeServiceFactory;
+import org.dspace.authorize.service.AuthorizeService;
+import org.dspace.builder.CommunityBuilder;
+import org.junit.Test;
+
+public class ContextModeIT extends AbstractIntegrationTestWithDatabase {
```

Comment:
I feel we should rename this to be `ContextIT` as it's the building block for additional ITs for the Context.java class (if necessary).